1 - identify organisations in the corpus/ner_flair.ipynb

```diff
@@ -1,25 +1,59 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    sf = open('../data-files/studies_on_water_scraped.json')
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/disaster-capitalism/named-entities/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
okay changed to `with` open and used os.path.